### PR TITLE
Feature/20299 pupil check audible timer

### PR DESCRIPTION
**Pull request by jon-shipley**
Add accessibility features for when the accessibility flag is set

*  audible beep 2 seconds before question timer ends
*  a different beep to signify the end of the question

### CODE REVIEW

File: pupil-spa/src/app/sound/sound.component.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
...AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAP/7ksD/gAAAASwAAAAAAAAlgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAD/+5LA/4AAAAEsAAAAAAAAJYAAAAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAA"
```
Comment (GeorgeChatzigiannis) [inline pupil-spa/src/app/sound/sound.component.html]: 🥇

**Comment by jon-shipley** [comment]
Looking for 1 review please.

File: pupil-spa/src/app/practice-question/practice-question.component.ts [hunk 2]
```diff
@@ -98,6 +99,19 @@ export class PracticeQuestionComponent implements OnInit, AfterViewInit {
     this.startTimer();
   }
 
+  /**
+   * Hook that runs before the timeout event (sent when the timer reaches 0 seconds)
+   */
+  preSendTimeoutEvent() {
```
Comment (GeorgeChatzigiannis) [inline pupil-spa/src/app/practice-question/practice-question.component.ts]: is that supposed to be an empty function?

Comment (GeorgeChatzigiannis) [reply pupil-spa/src/app/practice-question/practice-question.component.ts]: Oh I see the spoken practice question component is extending this

**Pull request merged**